### DISCUSSION TITLE
Like the new Wiki

### DISCUSSION THREAD

**Discussion by jkenzer** [jamessimone/apex-rollup#143] [General]
Can't wait to see it built out more. Things were getting a little too complex for a single readme file. Thanks @jamessimone

**Comment by jamessimone** [comment]
Adding more digestible documentation will be my focus, for the most part. Glad you like it!

**Comment by jamessimone** [comment]
Finally have had some time this past week to start back up work on the Wiki - looking forward to continuing to make progress there!
